### PR TITLE
Add bl + config for Nucleo H743ZI2.

### PR DESCRIPTION
**Pull request by JammuKekkonen**

### CODE REVIEW

**Comment by iot-pdmc** [comment]
Thanks for your changes.
@ARMmbed/pdmc-integrators please review.

**Comment by yogpan01** [comment]
@JammuKekkonen This should proabably be rebased since 5.15 is already merged.

**Comment by teetak01** [comment]
build-only

**Comment by iot-pdmc** [comment]
# [Test run: FAILED](https://jenkins-client.isgtesting.com/job/mbed-os-example-pelion/518/)
***

**Failed to build following targets:**
EP_AGORA-GCC_ARM-TCP
EP_AGORA-ARM-TCP
NUCLEO_H743ZI2-ARM-TCP
CY8CPROTO_062_4343W-ARM-TCP
CY8CPROTO_062_4343W-GCC_ARM-TCP
[Build artifacts](http://pdmc-ci.s3-website-eu-west-1.amazonaws.com/?prefix=jenkins-ci/mbed-os-example-pelion/518)

posted-by-tag: https://jenkins-client.isgtesting.com/job/mbed-os-example-pelion/

**Comment by teetak01** [comment]
build-only

**Comment by iot-pdmc** [comment]
# [Test run: FAILED](https://jenkins-client.isgtesting.com/job/mbed-os-example-pelion/521/)
***

**Failed to build following targets:**
NUCLEO_H743ZI2-ARM-TCP
[Build artifacts](http://pdmc-ci.s3-website-eu-west-1.amazonaws.com/?prefix=jenkins-ci/mbed-os-example-pelion/521)

posted-by-tag: https://jenkins-client.isgtesting.com/job/mbed-os-example-pelion/

**Comment by JammuKekkonen** [comment]
build-only

**Comment by iot-pdmc** [comment]
# [Test run: FAILED](https://jenkins-client.isgtesting.com/job/mbed-os-example-pelion/524/)
***

**Failed to build following targets:**
NUCLEO_H743ZI2-ARM-TCP
[Build artifacts](http://pdmc-ci.s3-website-eu-west-1.amazonaws.com/?prefix=jenkins-ci/mbed-os-example-pelion/524)

posted-by-tag: https://jenkins-client.isgtesting.com/job/mbed-os-example-pelion/

**Comment by JammuKekkonen** [comment]
build-only

**Comment by iot-pdmc** [comment]
# [Test run: SUCCESS](https://jenkins-client.isgtesting.com/job/mbed-os-example-pelion/525/)
***
[Build artifacts](http://pdmc-ci.s3-website-eu-west-1.amazonaws.com/?prefix=jenkins-ci/mbed-os-example-pelion/525)

posted-by-tag: https://jenkins-client.isgtesting.com/job/mbed-os-example-pelion/

File: README.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ Board                          |  Connectivity     | Storage for credentials and
 `Ublox UBLOX_C030_U201`             | Cellular          | SD card                 | Build-only
 `Ublox UBLOX_C030_R412M`            | Cellular          | SD card                 | Build-only
 `Embedded Planet EP_AGORA`          | Cellular          | SPIF                    | Build-only
+`ST NUCLEO_H743ZI2`                 | Ethernet          | Internal Flash          |
```
Comment (teetak01) [inline README.md]: Build-only ?

**Comment by teetak01** [comment]
build-only

**Comment by iot-pdmc** [comment]
# [Test run: SUCCESS](https://jenkins-client.isgtesting.com/job/mbed-os-example-pelion/526/)
***
[Build artifacts](http://pdmc-ci.s3-website-eu-west-1.amazonaws.com/?prefix=jenkins-ci/mbed-os-example-pelion/526)

posted-by-tag: https://jenkins-client.isgtesting.com/job/mbed-os-example-pelion/

**Pull request merged**